Sources/Errors/AlertCoordinator.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  AlertCoordinator.swift
+//  FuntastyKit
+//
+//  Created by Milan Strnad on 03/03/17.
+//  Copyright © 2017 The Funtasty s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public final class AlertCoordinator {
+
+    let parentViewController: UIViewController
+    weak var viewController: UIAlertController?
+
+    var error: Error?
```

Comment:
I don't like all these internal states. I think they could be handled much better since each init uses almost all different properties and the internal state must be error OR strings. What about enum?